enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Notebook.tsx

```diff
@@ -1,46 +1,72 @@
+import { LoadingOverlay } from "@mantine/core";
```

Comment:
I think we should reexport it instead of importing from `@mantine/core` directly like this. We should only allow importing `@mantine/core` within `metabase/ui` directory.

Reply:
yeah, this was more of a thing to play with when i first added it. we shouldn't use mantine imports for components